docs/quick_start_guide.md

```diff
@@ -180,11 +177,17 @@ $ docker plugin enable hpe
 >version=2.1
 
 ```
-$ docker plugin install store/hpestorage/hpedockervolumeplugin:<version> –-disable –-alias hpe
+$ docker plugin install store/hpestorage/hpedockervolumeplugin:2.1 –-disable –-alias hpe
 $ docker plugin set hpe glibc_libs.source=/lib64 certs.source=/tmp
 $ docker plugin enable hpe
 ```
 
+>version=3.0
```

Comment:
Please do not maintain all versions of plugin in readme. Keep it as generic instruction for all versions.